components/flipper.py

```diff
@@ -2,18 +2,28 @@
 
 
 class Flipper:
+    #this flips a bucket with an arm that has a spinning motor
+    
 
     flipper_motor = wpilib.Talon
+    # a talon controls a motor
     spinning = False
+    #It starts not spinning
 
     def turn_on(self):
+        '''defines "on"  '''
```

Comment:
Better wording might be something like "Sets flipper to on", although these kinds of comments are often not necessary. Since this function does exactly what is says (turn on the flipper) you could also remove it.

Reply:
It's Python convention to use double rather than single quotes for a Python docstring, so start and end with `"""`